openslides_backend/action/actions/poll/stop.py

```diff
@@ -36,8 +38,98 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             [
                 "poll_couple_countdown",
                 "poll_countdown_id",
+                "users_enable_vote_weight",
             ],
         )
         if meeting.get("poll_couple_countdown") and meeting.get("poll_countdown_id"):
             self.control_countdown(meeting["poll_countdown_id"], "reset")
+
+        # calculate votescast, votesvalid, votesinvalid
+        voted_ids = poll.get("voted_ids", [])
+        instance["votescast"] = f"{len(voted_ids)}.000000"
```

Comment:
Please cast the number to decimal and then use the string representation instead of building the string yourself

Reply:
Use another way to calculate votescast using decimal.